XMonad/Layout/IndependentScreens.hs

```diff
@@ -29,7 +29,7 @@ module XMonad.Layout.IndependentScreens (
     -- * Converting between virtual and physical workspaces
     -- $converting
     marshall, unmarshall, unmarshallS, unmarshallW,
-    marshallWindowSpace, unmarshallWindowSpace, marshallSort,
+    marshallWindowSpace, unmarshallWindowSpace, marshallSort
```

Comment:
It's better to have the trailing comma there, actually, it makes future git diffs smaller.